searchlib/src/vespa/searchlib/transactionlog/translogserver.cpp

```diff
@@ -144,6 +167,9 @@ TransLogServer::TransLogServer(const vespalib::string &name, int listenPort, con
         throw std::runtime_error(make_string("Failed creating tls base dir %s r(%d), e(%d). Requires manual intervention.", _baseDir.c_str(), retval, errno));
     }
     start(*_threadPool);
+    double chunkAgeLimit = vespalib::to_s(_domainConfig.getChunkAgeLimit());
+    auto staleCommitTask = new StaleCommitTask(_supervisor->GetScheduler(), *this, chunkAgeLimit);
+    staleCommitTask->ScheduleNow();
```

Comment:
This looks like a memory leak. You'll need to remember the task, and call Kill() as part of teardown.

Reply:
Plugged